CONTRIBUTING.md

```diff
@@ -37,8 +34,8 @@ make tests			# Application tests
 ### General ###
 - All new code must be properly tested: via unit tests (based on the Gtest framework; see `$REPO/unitTest`) or system tests (scripts exercising the main exiv2 application; see `$REPO/test`).
 - Code should be simple to read and to understand.
-- Do not invoke undefined behavior! Ensure that with UBSAN, i.e. compile your code with `-fsanitize=undefined` and run the test suite.
-- Ensure that your code has no memory errors. Use ASAN for that, i.e. compile your code with `-fsanitize=address`.
+- [Optional] Try to avoid undefined behavior. Ensure that with UBSAN, i.e. compile your code with `-fsanitize=undefined` and run the test suite.
+- [Optional] Ensure that your code has no memory errors. Use ASAN for that, i.e. compile your code with `-fsanitize=address`.
```

Comment:
No memory errors are not optional, we have enough security issues, we don't want new ones. ASAN can stay optional for MSVC users.

Reply:
changed